src/com/elixr/training/Main.java

```diff
@@ -58,29 +75,82 @@ public static String readFile(String fileName) {
     }
 
     public static boolean isSupportedFile(File file) {
-        if (file.getName().endsWith(TXTFILE) || file.getName().endsWith(JSONFILE) ) {
+        if (file.getName().endsWith(TXTFILE) || file.getName().endsWith(JSONFILE)) {
             System.out.println("File format supported");
             return true;
         } else {
             System.out.println("File format not supported");
             return false;
         }
     }
-    public static void searchWord(String data, String searchword) {
+
+    public static void searchWord(String inputFilePath, String data, String searchWord) throws SQLException {
         StringTokenizer st = new StringTokenizer(data);
         int count = 0;
         while (st.hasMoreTokens()) {
-            if (searchword.equalsIgnoreCase(st.nextToken())) {
+            if (searchWord.equalsIgnoreCase(st.nextToken())) {
                 count++;
             }
         }
         if (count == 0) {
             System.out.println("Word not found");
+            JDBCconnection.SearchFailure(inputFilePath, searchWord, count);
         } else {
             System.out.println("The word has been found");
             System.out.println("The word has been repeated for " + count + " times");
+            JDBCconnection.SearchSuccess(inputFilePath, searchWord, count);
+
         }
     }
+
+    class JDBCconnection extends Main {
```

Comment:
Different classes should be in different files

Reply:
done with the changes